dashboards/resources.libsonnet

```diff
@@ -197,6 +197,81 @@ local g = import 'grafana-builder/grafana.libsonnet';
         )
       ) + { tags: $._config.grafanaK8s.dashboardTags },
 
+  'k8s-resources-node.json':
```

Comment:
Do you have a screenshot of this dashboard at hand? Would be nice to visually see it.
But judging from the code here, I think it's a good addition! :slightly_smiling_face:

Reply:
I have add images to description.